### PR TITLE
Feat/plugin helper custom eggs

### PR DESCRIPTION
**Pull request by SushiRoll53**
Implemented whole Custom egg

### CODE REVIEW

File: lib/eggs/custom_egg.dart [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  @override
+  void alignChild() {
+    var tempNode = InjectedContainer(
+      child.bottomRightCorner,
+      child.topLeftCorner,
+      child.name,
+      child.UUID,
+    )..addChild(child);
+
+    getAttributeNamed('title').attributeNode = tempNode;
```
Comment (ivan-015) [inline lib/eggs/custom_egg.dart]: I believe the attribute name should be `child` 🤔

Comment (SushiRoll53) [reply lib/eggs/custom_egg.dart]: you are correct

Comment (ivan-015) [reply lib/eggs/custom_egg.dart]: Actually, sorry. I looked more closely at the code, I'm not even sure if we should wrap custom eggs' children inside containers. Do you think we should align custom eggs at all? 🤔

**Comment by SushiRoll53** [comment]
This was manually merged into `feat/pbdl-interpret`

**Pull request closed without merging**